framework/sessions/__init__.py

```diff
@@ -77,6 +76,7 @@ def prepare_private_key():
 
 
 def get_session():
+    Session = apps.get_model('osf.Session')
```

Comment:
So long as there's a reason. I'd just rather do things the python way than the Django way if we can.